include/can/core/message_writer.hpp

```diff
@@ -21,14 +24,15 @@ class MessageWriter {
      * @param node The node id
      * @param message The message to send
      */
-    template <message_core::Serializable Serializable>
-    void write(can_ids::NodeId node, const Serializable& message) {
+    template <message_core::CanResponseMessage ResponseMessage>
```

Comment:
Sort of a nitpick.... but if all response messages have the node_id then I feel that it should be the first item in the buffer.